### PR TITLE
Add 6.2.0 release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release note for 6.2.0 release.

## Motivation and Context
Support 6.2.0 release

### CODE REVIEW

File: content/sensu-go/6.2/release-notes.md [hunk 1]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+**IMPROVEMENTS:**
+
+- ([Commercial feature][193]) Refactored entity limiter to ensure that warning messages about approaching a license's entity or entity class limit are now only displayed for users with `create` or `update` permissions for the license.
+- Adjusted the format for silenced entry dates and durations in sensuctl. For all silenced entries, the begin date is now listed in RFC 3339 format. For silenced entries that have not begun, the list displays the expiration date in RFC 3339 format. For silenced entires with no expiration date, the list displays `-1`. For silenced entries that have begun, the list displays the duration (e.g. 1m30s).
+- Sensuctl and sensu-backend now ask users to retype their passwords when creating a new password in interactive mode.
+
+**FIXES:**
+
+- ([Commercial feature][193]) Sensuctl no longer produces an error when SSL certificates for the Vault provider do not exist on the local system.
+- ([Commercial feature][193]) The [prune API][194] and its [sensuctl interface][195] now need less broad permissions to work.
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
- ([Commercial feature][193]) The [prune API][194] and its [sensuctl interface][195] now require less-broad permissions.
```

File: content/sensu-go/6.2/release-notes.md [hunk 2]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**FIXES:**
+
+- ([Commercial feature][193]) Sensuctl no longer produces an error when SSL certificates for the Vault provider do not exist on the local system.
+- ([Commercial feature][193]) The [prune API][194] and its [sensuctl interface][195] now require less-broad permissions.
+- Logs now consistently use `event_id` rather than `event_uuid`.
+- Sensuctl commands that only contain subcommands now exit with status code 46 when no arguments or incorrect arguments are given.
+- The sensuctl dump command now includes a description.
+- Sensuctl command descriptions now have consistent capitalization.
+- The `config-file` flag is no longer order-dependent.
```
Comment (cwjohnston) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
- Use of the `config-file` flag is no longer order-dependent.
```

File: content/sensu-go/6.2/release-notes.md [hunk 3]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- Added /health endpoint to agentd.
+- Checks now include the [`scheduler` attribute][197], which Sensu automatically populates with the type of scheduler that schedules the check.
+- Events now include the [`sequence` attribute][198], which the Sensu agent automatically sets at startup and increments by 1 at every successive check execution or keepalive event.
+- Added support for using environment variables to define the configuration file paths for the Sensu agent (`SENSU_CONFIG_FILE`) and backend (`SENSU_BACKEND_CONFIG_FILE`).
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][193]) Refactored entity limiter to ensure that warning messages about approaching a license's entity or entity class limit are now only displayed for users with `create` or `update` permissions for the license.
+- Adjusted the format for silenced entry dates and durations in sensuctl. For all silenced entries, the begin date is now listed in RFC 3339 format. For silenced entries that have not begun, the list displays the expiration date in RFC 3339 format. For silenced entires with no expiration date, the list displays `-1`. For silenced entries that have begun, the list displays the duration (e.g. 1m30s).
```
Comment (cwjohnston) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
- Adjusted the format for silenced entry dates and durations in sensuctl tabular-format output. For all silenced entries, the begin date is now listed in RFC 3339 format. For silenced entries that have not begun, the list displays the expiration date in RFC 3339 format. For silenced entires with no expiration date, the list displays `-1`. For silenced entries that have begun, the list displays the duration (e.g. 1m30s).
```

Comment (cwjohnston) [reply content/sensu-go/6.2/release-notes.md]: Ideally we would link to an explanation of RFC 3339, but the most obvious route of linking to Wikipedia winds up at https://en.wikipedia.org/wiki/ISO_8601 which might be even more confusing. 🤔

File: content/sensu-go/6.2/release-notes.md [hunk 4]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+See the [upgrade guide][1] to upgrade Sensu to version 6.2.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][193]) Added support for the `memberof` attribute for the [LDAP authentication provider][199].
+- ([Commercial feature][193]) Added the ability to exclude resource types when using sensuctl prune with the [--omit flag][200].
+- ([Commercial feature][193]) Added support for [round robin scheduling on PostgreSQL][201] instead of etcd.
+- ([Commercial feature][193]) Added support for OIDC authentication via [sensuctl configure][202].
+- The [/metrics API endpoint][196] now exposes build information as a Prometheus metric.
+- Added /health endpoint to agentd.
```
Comment (cwjohnston) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
- Added /health API endpoint to agent websocket transport.
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Looks like we are missing an entry related to https://github.com/sensu/sensu-docs/pull/2906/

File: content/sensu-go/6.2/release-notes.md (hunk 4 above)
Comment (hillaryfraley) [reply content/sensu-go/6.2/release-notes.md]: Good catch...don't know why I thought anyone would put together what the heck I was talking about there :D

File: content/sensu-go/6.2/release-notes.md (hunk 3 above)
Comment (hillaryfraley) [reply content/sensu-go/6.2/release-notes.md]: Yeah. There is also the [golang time page](https://golang.org/pkg/time/#pkg-constants), but there's no good heading for linking straight to RFC3339 (readers would have to dig)

File: content/sensu-go/6.2/release-notes.md [hunk 5]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.2.0 release notes
+
+**December 17, 2020** &mdash; The latest release of Sensu Go, version 6.2.0, is now available for download.
+
+**ADD RELEASE SYNOPSIS HERE**
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
The latest release of Sensu Go, version 6.2.0, is now available for download! Sensu Go 5.x and configuration management users rejoice: this release adds support for agent local configuration (i.e., agent.yaml) managed entities! Agent entities may now be managed exclusively by their agents when `sensu-agent` is started with the new `agent-managed-entity` configuration option. This makes it more straightforward to migrate from Sensu Go 5.x to 6.x, as existing agent entity management workflows (i.e., Puppet) will just work with the new option enabled! Note that you will not be able to edit agent-managed entities via the backend REST API or web UI. This release also includes significant feature enhancements including PostgreSQL backend round robin check scheduling for better reliability and consistency. Also, the prune API no longer requires cluster-wide permissions; users with limited permissions can put it to use in their namespaces!
```

File: content/sensu-go/6.2/release-notes.md [hunk 6]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.2.0 release notes
+
+**December 17, 2020** &mdash; The latest release of Sensu Go, version 6.2.0, is now available for download.
+
+The latest release of Sensu Go, version 6.2.0, is now available for download! Sensu Go 5.x and configuration management users rejoice: this release adds support for agent local configuration (i.e., agent.yaml) managed entities! Agent entities may now be managed exclusively by their agents when `sensu-agent` is started with the new `agent-managed-entity` configuration option. This makes it more straightforward to migrate from Sensu Go 5.x to 6.x, as existing agent entity management workflows (i.e., Puppet) will just work with the new option enabled! Note that you will not be able to edit agent-managed entities via the backend REST API or web UI. This release also includes significant feature enhancements including PostgreSQL backend round robin check scheduling for better reliability and consistency. Also, the prune API no longer requires cluster-wide permissions; users with limited permissions can put it to use in their namespaces!
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
The latest release of Sensu Go, version 6.2.0, is now available for download! Sensu Go 5.x and configuration management users rejoice: this release adds support for agent local configuration (i.e. agent.yaml) managed entities! Agent entities may now be managed exclusively by their agents when `sensu-agent` is started with the new `agent-managed-entity` configuration option. This makes it more straightforward to migrate from Sensu Go 5.x to 6.x, as existing agent entity management workflows like Puppet will just work with the new option enabled! Note that you will not be able to edit agent-managed entities via the backend REST API or web UI. This release also includes significant feature enhancements such as PostgreSQL backend round robin check scheduling for better reliability and consistency. Also, the prune API no longer requires cluster-wide permissions; users with limited permissions can put it to use in their namespaces!
```

File: content/sensu-go/6.2/release-notes.md [hunk 7]
```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.2.0 release notes
+
+**December 17, 2020** &mdash; The latest release of Sensu Go, version 6.2.0, is now available for download.
+
+The latest release of Sensu Go, version 6.2.0, is now available for download! Sensu Go 5.x and configuration management users rejoice: this release adds support for agent local configuration (i.e. agent.yaml) managed entities! Agent entities may now be managed exclusively by their agents when `sensu-agent` is started with the new `agent-managed-entity` configuration option. This makes it more straightforward to migrate from Sensu Go 5.x to 6.x, as existing agent entity management workflows like Puppet will just work with the new option enabled! Note that you will not be able to edit agent-managed entities via the backend REST API or web UI. This release also includes significant feature enhancements such as PostgreSQL backend round robin check scheduling for better reliability and consistency. Also, the prune API no longer requires cluster-wide permissions; users with limited permissions can put it to use in their namespaces!
```
Comment (cwjohnston) [inline content/sensu-go/6.2/release-notes.md]:
```suggestion
The latest release of Sensu Go, version 6.2.0, is now available for download! Sensu Go 5.x and configuration management users rejoice: this release adds support for agent local configuration (i.e. agent.yml) managed entities! Agent entities may now be managed exclusively by their agents when `sensu-agent` is started with the new `agent-managed-entity` configuration option. This makes it more straightforward to migrate from Sensu Go 5.x to 6.x, as existing agent entity management workflows like Puppet will just work with the new option enabled! Note that you will not be able to edit agent-managed entities via the backend REST API or web UI. This release also includes significant feature enhancements such as PostgreSQL backend round robin check scheduling for increased reliability and consistency. Also, the prune API no longer requires cluster-wide permissions; users with limited permissions can put it to use in their namespaces!
```

agent.yaml -> agent.yml

better -> increased

**Review by cwjohnston — CHANGES_REQUESTED** [review]
The synopsis mentions agent-managed-entity and using prune without cluster-wide permissions, but I don't see corresponding entries under new, improved or fixed headings.

**Comment by hillaryfraley** [comment]
@cwjohnston I added some info about other improvements and some bug fixes. Wdyt?

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**